python/pyspark/sql/dataframe.py

```diff
@@ -5948,6 +5948,29 @@ def drop_duplicates(self, *subset: Union[str, List[str]]) -> "DataFrame":
 
         .. versionchanged:: 4.0.0
             Supports variable-length argument
+
+        Examples
```

Comment:
We have two examples for `drop duplicates,` and I think we can add more examples there.
Also cc @HyukjinKwon do we want to encourage users to use `drop_duplicates`?

Reply:
Nah, I wouldn't add a duplicated example. We already said it's an alias on the top.

Reply:
Thanks for weighing in @HyukjinKwon. I'll close this pull request. :)